### PR TITLE
DynamoDB Wrapper 💫

### PR DESCRIPTION
**Pull request by revett**
### Problem

**payments-api** needs a DynamoDB wrapper for local development.

### Solution

- Add one.
- Move `mock_config.go` to shared directory.
- Use separate `_test` packages for testing.

### Notes

- **Remember** to tag your merge commit after merging 👍

### CODE REVIEW

**Comment by revett** [comment]
[[AW] Add dynamodb wrapper for PA](https://trello.com/c/R0xIzv9V/886-aw-add-dynamodb-wrapper-for-pa)

File: dynamodb/client_test.go [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+import (
+	"testing"
+
+	"github.com/aws/aws-sdk-go/service/dynamodb/dynamodbiface"
+	"github.com/vidsy/awswrappers/dynamodb"
+)
+
+type (
+	// MockSDKClient used to mock the DynamoDB client used by the wrapper.
```
Comment (stevenjack) [inline dynamodb/client_test.go]: Don't need comment

**Comment by revett** [comment]
@stevenjack updated, please re-review 👍

File: dynamodb/client_config_test.go [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+		t.Run("CreatesWithValidConfig", func(t *testing.T) {
+			clientConfig, _ := dynamodb.NewClientConfigFromKMSConfig(&awswrappers.MockConfig{})
+
+			if clientConfig == nil {
+				t.Fatalf("Expected new ClientConfig, got: %v", clientConfig)
+			}
+		})
+
+		t.Run("ReturnsErrorWhenInvalidConfig", func(t *testing.T) {
+			var errorCases = []struct {
```
Comment (stevenjack) [inline dynamodb/client_config_test.go]: Don't really need this struct or the for loop as it's only for one case..

**Pull request merged**